doc/Type/Backtrace.pod6

```diff
@@ -68,6 +68,69 @@ Defined as:
 
 Returns a list of L<Backtrace::Frame> objects for this backtrace.
 
+=head2 method summary
+
+Defined as:
+
+    method summary(Backtrace:D: --> Str:D)
+
+Returns a summary string representation of the backtrace, filtered
+by C<!.is-hidden && (.is-routine || !.is-setting)>.
+
+    sub inner { say Backtrace.new.summary }
+    sub outer { inner; }
+    outer;
+
+    #OUTPUT:
```

Comment:
I would moved the section out of comments, you can prepend the listing with e.g. `This program:` and then `prints:`, and then you can just show the output without it being commented out or showing `perl6 test.p6`.
Something like:
```
This program:
$code
results in:
$output-as-just-monospace-text-without-indentation
```

Reply:
Maybe this could have been a bit more explicit
```
This program
    $code
results in
=for code :lang<text>
$output
```